src/provider/element-templates/ElementTemplates.js

```diff
@@ -194,8 +209,87 @@ export default class ElementTemplates {
 
     this._commandStack.execute('propertiesPanel.camunda.changeTemplate', context);
 
+    this._eventBus.fire(`elementTemplates.${action}`, payload);
+
     return context.element;
   }
+
+  /**
+   * Remove template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  removeTemplate(element) {
+    const replace = this._injector.get('replace'),
+          eventBus = this._injector.get('eventBus');
+
+    eventBus.fire('elementTemplates.remove', { element });
+
+    const businessObject = getBusinessObject(element);
+
+    const type = businessObject.$type,
+          eventDefinitionType = getEventDefinitionType(businessObject);
+
+    const newBusinessObject = createBlankBusinessObject(element, this._injector);
+
+    replace.replaceElement(element, {
+      type: type,
+      businessObject: newBusinessObject,
+      eventDefinitionType: eventDefinitionType
+    });
+  }
+
+  /**
+   * Unlink template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  unlinkTemplate(element) {
+    this.applyTemplate(element, null);
```

Comment:
Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/935/files#r1224061460. Let's always return the new or changed element.